react/MuiCozyTheme/Menus/index.js

```diff
@@ -1,3 +1,121 @@
-import Menu from '@material-ui/core/Menu'
+import React, { Component } from 'react'
 
-export default Menu
+import MenuButton from '../../Button'
+import PropTypes from 'prop-types'
+
+import ClickAwayListener from '@material-ui/core/ClickAwayListener'
+import Grow from '@material-ui/core/Grow'
+import Popper from '@material-ui/core/Popper'
+import Paper from '@material-ui/core/Paper'
+import MenuList from '@material-ui/core/MenuList'
+import { withStyles } from '@material-ui/core/styles'
+
+const styles = {
+  paper: {
+    '& hr': {
+      margin: '8px 0',
+      border: 0,
+      borderTop: '1px solid #D6D8DA'
```

Comment:
```suggestion
      borderTop: '1px solid var(--silver)'
```

Reply:
Yeah I didn't really cleaned the CSS yet but I changed that anyway.